index.js

```diff
@@ -18,6 +19,9 @@ module.exports = {
     "comment-whitespace-inside": "always",
     "declaration-bang-space-after": "never",
     "declaration-bang-space-before": "always",
+    "declaration-block-no-duplicate-properties": [ true, {
```

Comment:
`eslint-config-stylelint` is for consistent conventions for repos *within the stylelint org*. So, feel free to swap in whatever config you use within the suitcss org.